src/app/api/quote/utils.ts

```diff
@@ -0,0 +1,25 @@
+export interface FlashMintQuoteRequest {
+  chainId: number
+  account: string
+  inputToken: string
+  outputToken: string
+  inputAmount: string
+  outputAmount: string
+  slippage: number
+}
+
+export async function getQuote(quoteRequest: Record<string, string>) {
+  const query = new URLSearchParams(quoteRequest).toString()
+  const url = `https://api-pr-74-jrj6.onrender.com/api/v2/quote?${query}`
```

Comment:
// FIXME: change url for production when API is merged.

Reply:
kubb 😥